tests/Browser/ManageOrdersTest.php

```diff
@@ -302,6 +302,24 @@ public function test_cancel_order()
         });
     }
 
+    public function test_update_order_status(): void
+    {
+        $this->createOrders();
+        $order = Order::factory()->create(['status' => DeliveryStatus::Processing]);
+        $awaitingPayment = DeliveryStatus::localisedValue(DeliveryStatus::AwaitingPayment->value);
+
+        $this->browse(function (Browser $browser) use($order, $awaitingPayment) {
+           $browser->loginAs($this->admin)->visit(route('manage.orders.order', ['id' => $order->id]))
+               ->waitFor('#status-select')
+               ->click('#status-select')
+               ->assertSee($awaitingPayment)
+               ->screenshot('testorder')
```

Comment:
Is deze screenshot nodig?

Reply:
Oh, vergeten weg te halen! En over de cards, ik weet ook niet precies wat er is gebeurd, maar misschien heb ik per ongeluk ergens iets aangepast. Ik zal er nog eens naar kijken.